exist-core/src/main/java/org/exist/util/serializer/HTML5Writer.java

```diff
@@ -38,13 +38,13 @@ public class HTML5Writer extends XHTML5Writer {
 
     /**
      * Holds the names of the attributes that are considered boolean
-     * according to http://www.w3.org/TR/html51/single-page.html.
-     * 
+     * according to {@link <a href="http://www.w3.org/TR/html51/single-page.html">...</a>}
```

Comment:
I like that javadoc is corrected while at it. I would like to see a useful link text instead of `...`.